api/src/Service/JWT/JWTService.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service\JWT;
+
+use App\Entity\User;
+use Firebase\JWT\JWT;
+
+class JWTService
+{
+    private string $rootDir;
+
+    public function __construct(string $rootDir)
+    {
+        $this->rootDir = $rootDir;
+    }
+
+    public function createNewJWT(User $user)
+    {
+        $privateKey = file_get_contents(sprintf('%s/config/jwt/private.pem', $this->rootDir));
+        $publicKey = file_get_contents(sprintf('%s/config/jwt/public.pem', $this->rootDir));
+
+        $kid = openssl_digest($publicKey, 'sha256');
+        $payload = [
+            'username' => $user->getEmail(),
+            'userId' => $user->getId(),
```

Comment:
So one thing I've been thinking about for JWT auth across services/products is where we have identifiers that are service specific, making them [URN](https://en.wikipedia.org/wiki/Uniform_Resource_Name)s so that we can pass down chain and still have valid auth provided by another service. Something like `urn:opg:users:digideps:123`

Reply:
Yeah, where we have shared services like a PDF generator, say, it would allow a token generated by either Sirius or DigiDeps to be valid.